serenata_toolbox/federal_senate/dataset.py

```diff
@@ -42,9 +50,9 @@ def translate(self):
             csv_path = os.path.join(self.path, filename)
             try:
                 self._translate_file(csv_path)
-            except Exception as exception:
-                print('While translating Seranata Toolbox not found file: {} \n{}'.format(csv_path, exception))
-                not_found_files.append(csv_path)
+            except FileNotFoundError as file_not_found_error:
+                print('While translating Seranata Toolbox not found file: {} \n{}'.format(csv_path, file_not_found_error))
```

Comment:
Same as before: `'While translating, Serenata Toolbox didn't find the file: {} \n{}'`